bin/varnishd/cache/cache_req_fsm.c

```diff
@@ -104,6 +104,74 @@ cnt_transport(struct worker *wrk, struct req *req)
  * Deliver an object to client
  */
 
+int
+Resp_Setup(struct req *req, unsigned method)
```

Comment:
I think this should be two separate functions, the two callers share only a single source-line.